cpp/src/plasma/store.cc

```diff
@@ -126,23 +170,14 @@ const PlasmaStoreInfo* PlasmaStore::GetPlasmaStoreInfo() { return &store_info_;
 
 // If this client is not already using the object, add the client to the
 // object's list of clients, otherwise do nothing.
-void PlasmaStore::AddToClientObjectIds(const ObjectID& object_id, ObjectTableEntry* entry,
-                                       Client* client) {
+void PlasmaStore::AddToClientObjectIds(
+    const ObjectID& object_id, ObjectTableEntry* entry,
+    const std::shared_ptr<PlasmaClientConnection>& client) {
   // Check if this client is already using the object.
-  if (client->object_ids.find(object_id) != client->object_ids.end()) {
+  if (client->ObjectIDExists(object_id)) {
     return;
   }
-  // If there are no other clients using this object, notify the eviction policy
-  // that the object is being used.
-  if (entry->ref_count == 0) {
```

Comment:
Oops, where did this code go?

Reply:
I have move it into `IncreaseObjectRefCount` since there's a similar function in client side (`IncreasementObjectCount`).